kura/org.eclipse.kura.api/META-INF/MANIFEST.MF

```diff
@@ -29,13 +29,13 @@ Export-Package: org.eclipse.kura;version="1.3.0",
  org.eclipse.kura.gpio;version="1.0.1",
  org.eclipse.kura.linux.udev;version="1.0.0",
  org.eclipse.kura.message;version="1.1.0",
- org.eclipse.kura.net;version="1.2.1",
+ org.eclipse.kura.net;version="1.2.2",
```

Comment:
Ok, so after reading through the OSGi document again, I have to correct myself. *If* the API is considered to be implemented by "providers", then a minor increment would be sufficient. However in this case also the provider's import must be adapted to this, limiting the import to the minor version. Currently this is `[1.1,2.0)`.

However I also think that this API should not be considered "@noimplement", as the whole point of Kura is to be a modular framework, which allows users/consumers to replace modules with different implementations.

So yes, you are right, the minor version bump is enough. But the provider has to be adapted accordingly. Probably a problem with other Kura provider implementations as well. However this has to be documented. Like already described in [1], this interface should be annotated with e.g. `@ProviderType`.

Maybe it would be a good idea to start providing an "API policy" document like other Eclipse projects do, e.g. WTP [2], which is a bit more project specific than the page [1].

[1] https://github.com/eclipse/kura/wiki/Kura-Semantic-Versioning
[2] https://wiki.eclipse.org/WTP_API_Policy